### ISSUE TITLE
Fedora rawhide: test_metadata fails with cairo 1.17.4 (but passes with 1.16.0)

### ISSUE THREAD

**Issue by FelixSchwarz** [Kozea/cairocffi#177]
The test suite does not pass in Fedora rawhide which comes with cairo 1.17.4:

    @pytest.mark.xfail(cairo_version() < 11504,
                       reason='Cairo version too low')
    def test_metadata():
        …
       assert b"/CreationDate (20130721234600+01'00)" in pdf_bytes

`pdf_bytes` contains:

    b"%PDF-1.5
    …
    << /Producer (cairo 1.17.4 (https://cairographics.org))
       /Title (title)
       /Author (author)
       /Subject (subject)
       /Keywords (keywords)
       /Creator (creator)
       /CreationDate (20130721234600+01'00')
       /ModDate (20130721234600Z)
    >>
    …
    "

Basically `/CreationDate` now contains an additional `'` which I think is a valid PDF "date string".

**Comment by FelixSchwarz** [comment]
I suspect this is due to a cairo change in 1.17.4 and might  be a regression. I added a [comment in cairo PR 35](https://gitlab.freedesktop.org/cairo/cairo/-/issues/392#note_742384).
